src/components/Pressroom/Timeline/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Button, Step, StepIcon, StepLabel, Stepper, type Theme, Typography, useMediaQuery } from '@mui/material'
+import MobileStepper from '@mui/material/MobileStepper'
+import { useMemo, useState } from 'react'
+import LessThanIcon from '@/public/images/less-than.svg'
+import GreatThanIcon from '@/public/images/great-than.svg'
+import FullCircleIcon from '@/public/images/circle-full.svg'
+import EmptyCircleIcon from '@/public/images/circle-empty.svg'
+import { type Entry } from 'contentful'
+import { type TypeSimpleBaseBlockSkeleton } from '@/contentful/types'
+import css from './styles.module.css'
+
+const StepCard = ({ title, text }: { title: string; text: string }) => (
+  <div className={css.card}>
+    <Typography variant="h4" mb="16px">
+      {title}
+    </Typography>
+    <Typography>{text}</Typography>
+  </div>
+)
+
+type TimelineProps = { items: Entry<TypeSimpleBaseBlockSkeleton, undefined, string>[] }
+
+const Timeline = ({ items }: TimelineProps) => {
+  const [activeStep, setActiveStep] = useState(0)
+  const isSmallScreen = useMediaQuery((theme: Theme) => theme.breakpoints.down('sm'))
+  const STEPS_PER_DOT = useMemo(() => (isSmallScreen ? 1 : 3), [isSmallScreen])
```

Comment:
If you want to mark this as a constant, I'd move the values to the top of the file and use them here, e.g.

```suggestion
  const stepsPerDot = useMemo(() => (isSmallScreen ? STEPS_PER_DOT_SM : STEPS_PER_DOT_LG), [isSmallScreen])
```